healthvault-ios-sdk/Classes/Controllers/MHVBrowserAuthBroker.m

```diff
@@ -91,85 +94,84 @@ - (void)showAuthenticationControllerFromViewController:(UIViewController *)viewC
             vc = [[[UIApplication sharedApplication] keyWindow] rootViewController];
         }
         
-        UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:self];
+        if (!self.browserController)
```

Comment:
It does do that.  It sets browserController to nil when it completes and calls the completion block.  It doesn't create if it is set so test cases can setup a mock browser controller.  I didn't want a test init that would be confusing for people using the SDK